src/main/java/uk/gov/hmcts/reform/sscs/services/refdata/RegionalProcessingCenterService.java

```diff
@@ -1,96 +1,95 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import javax.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.StringUtils;
-import org.slf4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.core.io.ClassPathResource;
 import org.springframework.stereotype.Service;
 import uk.gov.hmcts.reform.sscs.exceptions.RegionalProcessingCenterServiceException;
 import uk.gov.hmcts.reform.sscs.models.refdata.RegionalProcessingCenter;
 
 @Service
+@Slf4j
 public class RegionalProcessingCenterService {
-    private static final Logger LOG = getLogger(RegionalProcessingCenterService.class);
 
     private static final String RPC_DATA_JSON = "reference-data/rpc-data.json";
     private static final String CSV_FILE_PATH = "reference-data/sscs-venues.csv";
     private static final char SEPARATOR_CHAR = '/';
     private static final String SSCS_BIRMINGHAM = "SSCS Birmingham";
 
-    private Map<String, RegionalProcessingCenter>  regionalProcessingCenterMap  = newHashMap();
-    private final Map<String, String> sccodeRegionalProcessingCentermap = newHashMap();
-    private final Map<String, String> venueIdToRegionalProcessingCentre = new HashMap<>();
+    private Map<String, RegionalProcessingCenter> regionalProcessingCenterMap = newHashMap();
+    private final Map<String, String> scCodeRegionalProcessingCenterMap = newHashMap();
+    private final Map<String, String> venueIdToRegionalProcessingCentre = newHashMap();
 
     private final AirLookupService airLookupService;
 
     @Autowired
-    public RegionalProcessingCenterService(AirLookupService airLookupService) {
+    RegionalProcessingCenterService(AirLookupService airLookupService) {
         this.airLookupService = airLookupService;
     }
 
 
     @PostConstruct
-    public void init() {
-        loadSccodeRpcMetadata();
+    void init() {
+        loadScCodeRpcMetadata();
         populateRpcMetadata();
     }
 
-    private void loadSccodeRpcMetadata() {
+    private void loadScCodeRpcMetadata() {
         ClassPathResource classPathResource = new ClassPathResource(CSV_FILE_PATH);
         try (CSVReader reader = new CSVReader(new InputStreamReader(classPathResource.getInputStream()))) {
 
             List<String[]> linesList = reader.readAll();
 
             linesList.forEach(line -> {
-                sccodeRegionalProcessingCentermap.put(line[1], line[2]);
+                scCodeRegionalProcessingCenterMap.put(line[1], line[2]);
                 venueIdToRegionalProcessingCentre.put(line[0], line[2]);
             });
         } catch (IOException e) {
-            LOG.error("Error occurred while loading the sscs venues reference data file: " + CSV_FILE_PATH,
-                new RegionalProcessingCenterServiceException(e));
+            throw new RegionalProcessingCenterServiceException(
```

Comment:
Is this correct? I thought the correct way for app insights to pick up errors was to Log the error and then throw the exception?